certify-mock.properties

```diff
@@ -1,226 +1,121 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
```

Comment:
Are we not removing this file ?

Reply:
this is the file for mock plugin use case properties

Reply:
then it has to be renamed to certify-mock-identity.properties